RSignatureVerifier/Environment.swift

```diff
@@ -0,0 +1,68 @@
+internal protocol EnvironmentSetupProtocol {
```

Comment:
Can it renamed as EnvironmentSettable or EnvironmentSetting?

Reply:
Done. (For me the most natural name in this case would be `EnvironmentType` but it doesn't match naming convention)